app/charts/pie/pie.tsx

```diff
@@ -19,7 +20,11 @@ export const Pie = () => {
   const [, dispatch] = useInteraction();
   const ref = useRef<SVGGElement>(null);
 
-  const maxSide = Math.min(chartWidth, chartHeight) / 2;
+  const isMobile = useIsMobile();
+
+  const maxSide = isMobile
+    ? Math.min(chartWidth, chartHeight)
+    : Math.min(chartWidth, chartHeight) / 2;
```

Comment:
@noahonyejese I think this is causing a bug that makes the pie chart grow beyond the available space, see the screenshot (you can reproduce by reducing the window size to a mobile-desktop breakpoint and seeing that it's correct on desktop, but broken just before the desktop breakpoint).

<img width="693" alt="Screenshot 2024-11-25 at 12 23 55" src="https://github.com/user-attachments/assets/eb514d76-cf6d-43e7-9c60-47526a6997f5">

I think it would be safest to revert this change, as handling this correctly would probably mean playing around with chart's aspect ratio, or some other related things higher up in the code hierarchy 👍